resources/asia/osm-asia-mailing-list.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "OSM-Asia-mailinglist",
+  "featureId": "asia_full",
```

Comment:
I think there is no `asia_full` geojson feature to link to.. Should we add one?

Reply:
Yes..

Reply:
Will you be adding it, or do you want me to?

Reply:
I added it in 7665810